modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -124,17 +135,17 @@ function stanford_profile_styles_node_form_wide(array $form, FormStateInterface
 
 /**
  * Implements hook_preprocess_HOOK().
- * for adding grid classes to multirow paragraphs
  */
-function stanford_profile_styles_preprocess_react_paragraphs(&$variables)
-{
+function stanford_profile_styles_preprocess_react_paragraphs(&$variables) {
+  // For adding grid classes to multirow paragraphs.
   foreach ($variables['rows'] as &$row) {
     foreach ($row['items'] as &$item) {
       $width = $item['width'];
       $item['attributes']->addClass("flex-$width-of-12");
     }
     $count = count($row['items']);
     $row['attributes']->addClass("container-$count-items");
+    $row['attributes']->addClass("flex-container");
```

Comment:
flex-container is needed to activate the flex-classes. This is new and wasn't required before because react_paragraphs set the container to `display:flex` but has since been removed.